frontend/components/atoms/FormEntry.js

```diff
@@ -11,7 +11,7 @@ export default function FormEntry({
   limitations,
 }) {
   return (
-    <Link href="#">
+    <Link href="/post/1">
```

Comment:
`/post/{id}`?

Reply:
Can't do that yet, I still have static data not everything is ready